package/sbp_protocol/src/sbp_router.yml

```diff
@@ -0,0 +1,112 @@
+
+name: SBP_ROUTER
+ports:
+  - name: SBP_PORT_FIRMWARE
+    pub_addr: "@tcp://127.0.0.1:43010"
+    sub_addr: "@tcp://127.0.0.1:43011"
+    forwarding_rules:
+      - dst_port: SBP_PORT_SETTINGS_DAEMON
+        filters:
```

Comment:
As I understand, YAML is a superset of JSON, so the `{ }` stuff is valid. I just wanted each rule on a single line for readability.